docs-client/src/lib/transports/transport.ts

```diff
@@ -47,7 +51,37 @@ export default abstract class Transport {
     return this.doSend(method, filledHeaders, bodyJson, endpointPath, queries);
   }
 
-  public abstract supportsMimeType(mimeType: string): boolean;
+  public curlPathAndBody(method: Method, body: string): [string, string] {
+    const endpoint = this.findDebugMimeTypeEndpoint(method);
+    return [
+      this.getCurlPath(endpoint),
+      this.getCurlBody(body, endpoint, method),
+    ];
+  }
+
+  protected findDebugMimeTypeEndpoint(method: Method): Endpoint {
+    const endpoint = method.endpoints.find((ep) =>
+      ep.availableMimeTypes.includes(this.getDebugMimeType()),
+    );
+    if (!endpoint) {
+      throw new Error(
+        `Endpoint does not support debug transport. MimeType: ${this.getDebugMimeType()}`,
+      );
+    }
+    return endpoint;
+  }
+
+  protected getCurlPath(endpoint: Endpoint): string {
+    return endpoint.pathMapping;
+  }
+
+  protected getCurlBody(
+    body: string,
+    _endpoint: Endpoint,
+    _method: Method,
```

Comment:
nit: Could remove underscores before the parameters?

Reply:
Since they are unused method parameters (an overridden method uses), they need to start w/ an underscore.

Reply:
Forgot that ESLInt has that rule. 😆